<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>refinements_editor</class>
 <widget class="QDialog" name="refinements_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Refinements Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="reset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="refinement_actions_group">
     <property name="title">
      <string>Refinement Actions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="action"/>
      </item>
      <item>
       <widget class="QPushButton" name="apply_action">
        <property name="text">
         <string>Apply Action</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QVBoxLayout" name="tree_view_layout"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>action</tabstop>
  <tabstop>apply_action</tabstop>
  <tabstop>reset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
